src/BuildOnWindows.cmake

```diff
@@ -40,8 +40,12 @@ if(BUILD_SEPARATE_OPS)
   endforeach()
 else()
   # Split SYCL kernels into 2 libraries as categories 1) Unary+Binary 2) Others.
-  set(ATen_XPU_SYCL_UNARY_BINARY_SRCS)
-  set(ATen_XPU_SYCL_NON_UNARY_BINARY_SRCS)
+  set(ATen_XPU_SYCL_BINARY_SRCS)
+  set(ATen_XPU_SYCL_UNARY_SRCS)
+  set(ATen_XPU_SYCL_REDUCE_SRCS)
+  set(ATen_XPU_SYCL_ACTIVATION_SRCS)
+  set(ATen_XPU_SYCL_FOREACH_SRCS)
+  set(ATen_XPU_SYCL_OTHERS_SRCS)
```

Comment:
@JianpingChen066 per our experience, separate kernels libraries according to AOT arch will be better than operators, for the binary size

Reply:
If splitting bin according to AOT arch, do you have any change on host? I cannot image how to call MTL add_kernel without changes on host.

Reply:
ok for stock pytorch, we don't have a device management so no dispatch available

Reply:
Yes. But I am not sure to uploading device management from runtime to fwk is reasonable so far. We expose the complexity.